src/templates/admin/elements/publish/author.html

```diff
@@ -32,4 +36,12 @@ <h4><i class="fa fa-envelope-o">&nbsp;</i>Notify Author</h4>
     <button class="close-button" data-close aria-label="Close reveal" type="button">
         <span aria-hidden="true">&times;</span>
     </button>
-</div>
\ No newline at end of file
+</div>
+<script type="module">
+    document.addEventListener('DOMContentLoaded', () => {
+        tinymce.init({
+            selector: '#notify_author_email',
+        })
+    })
+</script>
```

Comment:
I suspect initialising TinyMCE this way will discard the config coming from settings.py. If so, can we use a simple Django Form with just this field? It is probably worth doing so regardless of this concern in any case

Reply:
> this way will discard the config coming from settings.py

Yeah, it would not pick up the config, but it was the simplest way to patch it because of the view's entanglement with logic and the notify sequence. I've changed it to a Django form as discussed offline. Is this what you had in mind?